docs/one-click-integrations/bitbucket-integration.md

```diff
@@ -81,11 +110,25 @@ The table below lists the data that the Bitbucket integration collects from your
         <p>Deployments:</p>
         <ul>
             <li><code>deploy_id</code>: unique pull request identifier</li>
+            <li><code>timestamp_created</code>: pull request merged date</li>
             <li><code>system</code>: repository name</li>
         </ul>
     </td>
     <td>Deployment frequency and Change failure rate on the <a href="../../metrics/accelerate/">Accelerate Overview dashboard</a></td>
 </tr>
+<tr>
+    <td>Pull requests</td>
+    <td>
+        <p>Incidents:</p>
+        <ul>
+            <li><code>incident_id</code>: pull request number</li>
+            <li><code>timestamp_created</code>: reverted pull request merged date</li>
+            <li><code>timestamp_resolved</code>: pull request merged date</li>
```

Comment:
See the [comment on the related PR](https://github.com/codacy/pulse-user-docs/pull/195/files#r1108223041)

```suggestion
            <li><code>timestamp_created</code>: merge date of the reverted pull request</li>
            <li><code>timestamp_resolved</code>: merged date of the pull request</li>
```